spec/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/refresher_spec.rb

```diff
@@ -112,14 +112,13 @@ def assert_specific_flavor
     def assert_specific_vm
       instance_name = "test-instance-rhel-s922-shared-tier3"
       vm = ems.vms.find_by(:name => instance_name)
-      placement_group = ems.placement_groups.find_by(:name => "test-placement-group-affinity")
       expect(vm).to have_attributes(
         :location           => "unknown",
         :name               => instance_name,
         :description        => "PVM Instance",
         :vendor             => "ibm_power_vs",
         :power_state        => "on",
-        :placement_group_id => placement_group.id,
+        :placement_group_id => nil,
```

Comment:
Hey so my only concern here is that we're no longer testing the placement_group lazy find here https://github.com/ManageIQ/manageiq-providers-ibm_cloud/pull/431/files#diff-f985d1bd514a044797c4062d1d8d94ff3da33e97f6bfa8d182955661184e7d45R64
We should also be testing the resource group lazy find https://github.com/ManageIQ/manageiq-providers-ibm_cloud/pull/431/files#diff-f985d1bd514a044797c4062d1d8d94ff3da33e97f6bfa8d182955661184e7d45R65

Reply:
Good catch. I switched `assert_specific_vm` to use a VM that is still a member of a placement group so we can keep the lazy find coverage.

For the resource group lazy find we'll need to record another VCR cassette with a shared processor pool (the current cassette doesn't have any). fyi @miyamotoh